kind-cluster/install.sh

```
@@ -26,12 +26,12 @@ fi
 # ----------------------------
 if ! command -v kind &>/dev/null; then
   echo "📦 Installing Kind..."
-
+  KIND_VERSION="v0.31.0"
```

Shellcheck (0.11.0)

[warning] KIND_VERSION appears unused. Verify use (or export if used externally).

(SC2034)